jquery.visible.js

```diff
@@ -29,10 +29,12 @@
             var rec = t.getBoundingClientRect(),
                 tViz = rec.top    >= 0 && rec.top    <  vpHeight,
                 bViz = rec.bottom >  0 && rec.bottom <= vpHeight,
+                mVis = rec.top    <  0 && rec.bottom >  vpHeight,   
                 lViz = rec.left   >= 0 && rec.left   <  vpWidth,
                 rViz = rec.right  >  0 && rec.right  <= vpWidth,
-                vVisible   = partial ? tViz || bViz : tViz && bViz,
-                hVisible   = partial ? lViz || lViz : lViz && rViz;
+                hmVis= rec.left   <  0 && rec.right  >  vpWidth,
+                vVisible   = partial ? tViz || bViz || mVis : tViz && bViz,
+                hVisible   = partial ? lViz || lViz ||hmVis : lViz && rViz;
```

Comment:
:+1:  I just made this same change.  Didn't have to look at the non native implementation because I don't have to support browsers without getBoundingClientRect :)